packages/assistify-ai/client/chatpalAdoption.js

```diff
@@ -0,0 +1,8 @@
+TAPi18n.loadTranslations({
+	en: {
+		CHATPAL_ENTER_SEARCH_STRING: 'Search knowledge base'
```

Comment:
This was a try overwriting the i18-Definition without modifying it. However, this doesn't seem to work. Kept it for reference though